.travis.yml

```diff
@@ -19,8 +19,42 @@ addons:
       - make
       - unzip
 script:
+  - mkdir -p build && cd build
+  - cmake -DCMAKE_INSTALL_PREFIX=/rsc ..
+  - make
+  - make test
+  - make DESTDIR=install_dir install
+  - tar -C install_dir -czf "rsc-${CC}.tar.gz" rsc
   - |
-      mkdir -p build && cd build
-      cmake ..
-      make
-      make test
+    cat << EOF > bintray.json
+    {
+      "package": {
+          "name": "rsc",
+          "repo": "travis-artifacts",
+          "subject": "open-rsx",
+          "desc": "Travis artifact for branch ${TRAVIS_BRANCH}",
+          "vcs_url": "https://github.com/open-rsx/rsc.git",
+          "licenses": ["LGPL-3.0"]
+      },
+
+      "version": {
+          "name": "travis-${TRAVIS_BRANCH}",
+          "vcs_tag": "${TRAVIS_BRANCH}"
+      },
+
+      "files":
+          [
+              {"includePattern": "\./(rsc-.*).tar.gz", "uploadPattern": "\$1-${TRAVIS_BRANCH}.tar.gz", "matrixParams": {"override": 1}}
```

Comment:
Put pairs on separate lines?

Reply:
what pairs?

Reply:
JSON key-value pairs on the long line (line 47).

Reply:
This was formatted according to the example file for bintray, but I can change that.